cmd/acr-credential-provider/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
+
```

Comment:
Could you remove the new line and format the code please?

Reply:
the `local-prefixes: sigs.k8s.io/cloud-provider-azure` will group the local import, the new line is expected I think.